### PR TITLE
#20 : display pagination

### PR DESCRIPTION
**Pull request by landreaubis**

### CODE REVIEW

File: src/AppBundle/Controller/BlurayController.php [hunk 1]
```diff
@@ -14,8 +13,22 @@ class BlurayController extends Controller
...
     public function listingAction($page = 1, $maxperpage = 4)
     {
-        $listBluray = $this->getDoctrine()->getRepository('AppBundle:Bluray')->findAll();
-        return ['listBluray' => $listBluray];
+        $blurayRepository = $this->getDoctrine()->getRepository('AppBundle:Bluray');
+        $query = $blurayRepository->createQueryBuilder('b')
+            ->setFirstResult(($page-1) * $maxperpage)
+            ->setMaxResults($maxperpage)
+            ->getQuery()
+        ;
```
Comment (Jean-Beru) [inline src/AppBundle/Controller/BlurayController.php]: Put your query in a dedicated repository (eg. AppBundle\Repository\BlurayRepository, AppBundle\Entity\BlurayRepository or AppBundle\Entity\Repository\BlurayRepository)

See https://github.com/Jean-Beru/dunkerque/tree/master/src/AppBundle/Entity.

File: src/AppBundle/Controller/BlurayController.php [hunk 2]
```diff
@@ -14,8 +13,22 @@ class BlurayController extends Controller
      */
     public function listingAction($page = 1, $maxperpage = 4)
```
Comment (Jean-Beru) [inline src/AppBundle/Controller/BlurayController.php]: `$maxperpage` should be in your repository (or in your configuration file).

**Comment by landreaubis** [comment]
Hello Jean-Beru,

it's good for you ?

File: src/AppBundle/Controller/BlurayController.php [hunk 3]
```diff
@@ -14,8 +13,17 @@ class BlurayController extends Controller
...
     {
-        $listBluray = $this->getDoctrine()->getRepository('AppBundle:Bluray')->findAll();
-        return ['listBluray' => $listBluray];
+        $query = $this->getDoctrine()->getRepository('AppBundle:Bluray')->findAll($page, $maxperpage);
```
Comment (Jean-Beru) [inline src/AppBundle/Controller/BlurayController.php]: Why filter by page and maxperpage in `findAll` if paginator also does it?

File: src/AppBundle/Entity/BlurayRepository.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?php
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\EntityRepository;
+
+class BlurayRepository extends EntityRepository
+{
+    public function findAll ($page, $maxperpage) {
```
Comment (Jean-Beru) [inline src/AppBundle/Entity/BlurayRepository.php]: findAll is already a repository method (which returns all entities). Use a different method name... Or do not use it (see previous comment)

**Comment by Jean-Beru** [comment]
:+1: (squash & merge)

**Pull request merged**